make/migrations/postgresql/0011_1.10.0_schema.up.sql

```diff
@@ -0,0 +1,35 @@
+/*Table for keeping the plug scanner registration*/
+CREATE TABLE scanner_registration
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    uid VARCHAR(64) UNIQUE NOT NULL,
+    url VARCHAR(256) UNIQUE NOT NULL,
+    name VARCHAR(128) UNIQUE NOT NULL,
+    description VARCHAR(1024) NULL,
+    auth VARCHAR(16) NOT NULL,
+    access_cred VARCHAR(512) NULL,
+    adapter VARCHAR(128) NOT NULL,
```

Comment:
As for `adapter` `vendor` `version`, which can be retrieved via metadata API of the scanner adapter, we may not want to store them in DB, in out-of-tree scenario, the scanner adapter has independent lifecycle of Harbor, so it can be tricky when they are upgraded or switched to other scanner

Reply:
We have discussed this. I'd like to show a little more info to the users not just name and endpoint URL. This info is mostly like a cache of part of the metadata. We'll update it if there are any changes introduced when triggering scanner.

I'm planning to test the connection by calling the metadata API before launch the real job. Maybe the cache can be updated if there are changes.

Reply:
Keep it as it is now.

Reply:
After offline discussion, we agreed such dynamic info should not be tracked in Harbor DB.

Reply:
YES